portfolio/src/main/webapp/script.js

```diff
@@ -19,28 +19,55 @@
  * Fetches comments from the server and adds it to the DOM
  */
 function getComments() {
-    fetch("/data").then(response => response.json()).then((commentsData) => {
+    fetch("/comments").then(response => response.json()).then((commentsData) => {
         const commentsContainer = document.getElementById("comments-container");
         commentsContainer.innerHTML = '';
-        for (var i = 0; i < commentsData.length; i++) {
-            var commentData = commentsData[i];
+        for (const i = 0; i < commentsData.length; i++) {
+            const commentData = commentsData[i];
 
-            // Create new comment element
-            var boldElement = document.createElement("b");
-            var boldNode = document.createTextNode("Comment " + (i + 1) + " - ");
-            boldElement.appendChild(boldNode);
+            // Create new comment elements
+            const commentHeader = createCommentHeader(commentData);
+            const commentContent = createCommentContent(commentData);
 
-            var commentElement = document.createElement("h4");
-            var textNode = document.createTextNode(commentData.comment);
-            commentElement.appendChild(boldElement);
-            commentElement.appendChild(textNode);
-
-            // Add new comment element to comments section
-            commentsContainer.appendChild(commentElement);
+            // Add new comment to comments section
+            commentsContainer.appendChild(commentHeader);
+            commentsContainer.appendChild(commentContent);
         }
     });
+
+}
+
+/* Creates header for comment containing name in bold and timestamp in italics */
+function createCommentHeader(commentData) {
```

Comment:
Would prefer if these new changes are part of a new pull request in the future if the pull request is already in review.

Reply:
Could we document the input parameter and return parameter types in the comments: https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
Sorry, I will make sure to do that in the future. I have added the documentation in the latest commit. Thanks!